NEWS.md

```diff
@@ -6,6 +6,10 @@
     
 *   `coord_sf()` now respects manual setting of axis tick labels (@clauswilke,
     #2857, #2881).
+    
+*   `coord_sf()` now accepts two new parameters `label_graticule` and `label_axes`
```

Comment:
Because I'm the worst, can we offset the params as appositives? "…two new parameters, `label_graticule and `label_axes`,…"

Reply:
Will do.